pandas/computation/tests/test_eval.py

```diff
@@ -50,11 +50,7 @@ def _eval_single_bin(lhs, cmp1, rhs, engine):
         try:
             return c(lhs, rhs)
         except ValueError as e:
-            try:
-                msg = e.message
-            except AttributeError:
-                msg = e
-            msg = u(msg)
+            msg = u(str(e))
```

Comment:
Could you provide the issue number?  My computer is a little weird right now, so I can't really do much on it ATM (and searching GitHub not on computer is not convenient).